pkg/service/turbo_communication_config.go

```diff
@@ -32,6 +32,7 @@ type TurboCommunicationConfig struct {
 	mediationcontainer.ServerMeta      `json:"serverMeta,omitempty"`
 	mediationcontainer.WebSocketConfig `json:"websocketConfig,omitempty"`
 	RestAPIConfig                      `json:"restAPIConfig,omitempty"`
+	mediationcontainer.SdkProtocolConfig
```

Comment:
Final comment, don't we need a `json` annotation here?

Reply:
Added json annotiations